src/main/java/com/softserve/demo/util/OfferMapper.java

```diff
@@ -0,0 +1,37 @@
+package com.softserve.demo.util;
+
+import com.softserve.demo.dto.OfferDTO;
+import com.softserve.demo.model.Offer;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+
+import java.util.List;
+
+@Mapper(componentModel="spring")
+public interface OfferMapper {
+
+    @Mappings({
+            @Mapping(target = "id", source = "id"),
+            @Mapping(target = "description", source = "description"),
+            @Mapping(target = "startDate", source = "startDate"),
```

Comment:
If names have identical names. Map struct will map them anyway

Reply:
I fixed it.